.cirrus.yml

```diff
@@ -40,8 +40,12 @@ task:
     - fuzz_cov
   << : *GLOBAL_TASK_TEMPLATE
   build_script:
-    - cd bitcoin-core/build
-    - ../configure --enable-zmq --with-incompatible-bdb --enable-lcov --enable-lcov-branch-coverage --disable-bench --enable-extended-functional-tests
+    - cd bitcoin-core
+    - ./test/get_previous_releases.py -b -t ./build/releases
+    - cd build
+    - ../contrib/install_db4.sh `pwd`
+    - export BDB_PREFIX='/tmp/cirrus-ci-build/bitcoin-core/build/db4'
```

Comment:
looks like this is also changing bdb? Probably best to leave this as-is?

Reply:
It is required to compile BDB to run backward compatibility tests.

Reply:
bdb is already installed, no?

Reply:
it used to be configured with `--with-incompatible-bdb` but the tests couldn't run so I had to do this

Reply:
https://cirrus-ci.com/task/5800398698053632?logs=build#L3388
here's a run with `--with-incompatible-bdb`

> 131/252 - feature_backwards_compatibility.py --legacy-wallet failed, Duration: 7 s
stdout:
2022-10-06T21:58:26.472000Z TestFramework (INFO): Initializing test directory /tmp/test_runner_₿_🏃_20221006_213446/feature_backwards_compatibility_118
2022-10-06T21:58:29.279000Z TestFramework (INFO): Test wallet backwards compatibility...
2022-10-06T21:58:32.877000Z TestFramework (ERROR): JSONRPC error
Traceback (most recent call last):
  File "/tmp/cirrus-ci-build/bitcoin-core/test/functional/test_framework/test_framework.py", line 133, in main
    self.run_test()
  File "/tmp/cirrus-ci-build/bitcoin-core/build/../test/functional/feature_backwards_compatibility.py", line 186, in run_test
    node.loadwallet(wallet_name)
  File "/tmp/cirrus-ci-build/bitcoin-core/test/functional/test_framework/coverage.py", line 49, in __call__
    return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)
  File "/tmp/cirrus-ci-build/bitcoin-core/test/functional/test_framework/authproxy.py", line 144, in __call__
    raise JSONRPCException(response['error'], status)
test_framework.authproxy.JSONRPCException: Wallet file verification failed. Error initializing wallet database environment "/tmp/test_runner_₿_🏃_20221006_213446/feature_backwards_compatibility_118/node2/regtest/wallets/w1"! This error could occur if this wallet was not shutdown cleanly and was last loaded using a build with a newer version of Berkeley DB. If so, please use the software that last loaded this wallet (-4)